ui/src/app/dashboard/views/DashboardConfigurationForm/DashboardConfigurationSubnetForm/DashboardConfigurationSubnetForm.tsx

```diff
@@ -0,0 +1,115 @@
+};
+
+const DashboardConfigurationSubnetForm = (): JSX.Element => {
+  const dispatch = useDispatch();
+  const subnets = useSelector(subnetSelectors.all);
+  const fabrics = useSelector(fabricSelectors.all);
+  const subnetsLoaded = useSelector(subnetSelectors.loaded);
+  const fabricsLoaded = useSelector(subnetSelectors.loaded);
+  const saved = useSelector(subnetSelectors.saved);
+  const saving = useSelector(subnetSelectors.saving);
+
+  useEffect(() => {
+    dispatch(subnetActions.fetch());
+    dispatch(fabricActions.fetch());
+  }, [dispatch]);
+
+  const loaded = subnetsLoaded && fabricsLoaded;
+  let content: JSX.Element = <Spinner text="Loading..." />;
+
+  if (loaded) {
+    const initialValues: SubnetDiscoveryValues = {};
+    subnets.forEach((subnet) => {
+      initialValues[subnet.id] = subnet.active_discovery;
+    });
+
+    content = (
+      <FormikForm<SubnetDiscoveryValues>
+        buttonsAlign="left"
+        buttonsBordered={false}
+        initialValues={initialValues}
+        onSubmit={(values, { resetForm }) => {
+          subnets.forEach((subnet) => {
+            if (subnet.active_discovery !== values[subnet.id]) {
+              dispatch(
+                subnetActions.update({
+                  active_discovery: values[subnet.id],
+                  id: subnet.id,
+                })
+              );
+            }
+          });
+          resetForm({ values });
+        }}
+        resetOnSave
+        saving={saving}
+        saved={saved}
+      >
+        {subnets.map((subnet) => {
+          const targetFabric = fabrics.find((fabric) =>
+            fabric.vlan_ids.includes(subnet.vlan)
+          );
+          return (
+            <FormikField
+              key={`subnet-${subnet.id}`}
+              label={
+                <>
+                  <LegacyLink
+                    data-test="subnet-link"
+                    route={baseURLs.subnet({ id: subnet.id })}
+                  >
+                    {subnet.cidr}
+                  </LegacyLink>
+                  {targetFabric && (
+                    <>
+                      <span> on </span>
+                      <LegacyLink
+                        data-test="fabric-link"
+                        route={baseURLs.fabric({ id: targetFabric.id })}
+                      >
+                        {targetFabric.name}
+                      </LegacyLink>
+                    </>
+                  )}
+                </>
+              }
+              name={`${subnet.id}`}
+              type="checkbox"
+            />
+          );
+        })}
+      </FormikForm>
+    );
+  }
+
+  return (
+    <Strip shallow>
+      <h3 className="p-heading--4">Subnet mapping</h3>
+      <p>
+        Active discovery (subnet mapping) can be enabled below on the configured
+        subnets. Each rack will scan the subnets that allow it. This helps
+        ensure discovery information is accurate and complete.
+      </p>
```

Comment:
@petesfrench I've also added a little context about what the checkboxes actually mean (just lifted from the legacy version's help text)